### PR TITLE
feat(docs): upgrade readme 

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+| [`@mozaic-ds/tokens`](./packages/tokens)                 | Tokens are theming/styling constants defined in a JSON file. They are trasnpiled into multiple format to be shared across plateforms (SCSS, IOS, ANDROID...).                                                     |
+| [`@mozaic-ds/css-dev-tools`](./packages/css-dev-tools)     | Right now the tools are composed only of SCSS/PostCSS tools.                                                                                                                   |
+| [`@mozaic-ds/web-fonts`](./packages/web-fonts)         | All fonts available in MDS (LM fonts currently)                                                                                                                                                    |
+| [`@mozaic-ds/icons`](./packages/icons)           | Iconography assets. We also offer support in: [React](./packages/icons/react), [Ios](./packages/icons/pdf), and [Vue](./packages/icons/vue) |
+
+
+## :books: Documentation
 We tried to focus on the developper experience, meaning that working with it should be as simple as possible.
```
Comment (nherwegh) [inline README.md:44]: developer

**Review by nherwegh — CHANGES_REQUESTED** [review]

**Review by nherwegh — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
-
-Building a styleguide using a static site generator instead of relying on a database allow us to use a versionned styleguide that matches the assets and framework that it is based on.
-
-We tried to focus on the developper experience, meaning that working with it should be as simple as possible.
+<h1 align="center">
+  Mozaic Design System
+</h1>
+
+> Mozaic is an open-source design system built by Adeo. A design system to reinforce team’s delivery time, 
+> improve quality of deliverables and thus help set up a coherent customer experience.
```
Comment (tiloyi) [inline README.md:6]: > Mozaic is an open-source design system built by Adeo; to reinforce team’s delivery time, improve quality of deliverables and thus help set up a coherent customer experience.

File: README.md [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+</p>
+
+## Getting started
+If you are just getting started, check out our documentation: https://mozaic.adeo.cloud/
+
+If your are looking for JS components, we have community-contributed components for the following technologies: 
+- [Freemarker](https://github.com/adeo/mozaic-freemarker)
+- [Vue](https://github.com/adeo/mozaic-vue)
+- [Angular](https://github.com/adeo/mozaic-angular)
+- [React](https://github.com/adeo/mozaic-react)
```
Comment (tiloyi) [inline README.md:30]: Personally, I wouldn't mention the React version because nobody uses it.

File: README.md [hunk 4]
```diff
@@ -1,17 +1,47 @@
...
+
+If you're trying to find something specific, here's a list of our main packages that we support!
+
+| Package name                                  | Description                                                                                                                                                                             |
+| --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`@mozaic-ds/styles`](./packages/styles)                 | All the CSS and SCSS files that constitute the Mozaic SCSS framework.                                                                                                                                                 |
```
Comment (tiloyi) [inline README.md:36]: All the SCSS source files

File: README.md [hunk 5]
```diff
@@ -1,17 +1,47 @@
...
+| [`@mozaic-ds/tokens`](./packages/tokens)                 | Tokens are theming/styling constants defined in a JSON file. They are trasnpiled into multiple format to be shared across plateforms (SCSS, IOS, ANDROID...).                                                     |
```
Comment (tiloyi) [inline README.md:37]: ~~trasnpiled~~ => transpiled

File: README.md [hunk 6]
```diff
@@ -1,17 +1,47 @@
...
+| [`@mozaic-ds/css-dev-tools`](./packages/css-dev-tools)     | Right now the tools are composed only of SCSS/PostCSS tools.                                                                                                                   |
```
Comment (tiloyi) [inline README.md:38]: ~~Right now the tools are composed only of SCSS/PostCSS tools.~~
=> Tools for css like PostCSS plugins and linters config

File: README.md [hunk 7]
```diff
@@ -1,17 +1,47 @@
...
+| [`@mozaic-ds/web-fonts`](./packages/web-fonts)         | All fonts available in MDS (LM fonts currently)                                                                                                                                                    |
```
Comment (tiloyi) [inline README.md:39]: ~~LM fonts currently~~
=> Currently, the Leroy Merlin font only

**Pull request merged**